tempto-core/src/main/java/io/trino/tempto/internal/query/CassandraQueryExecutor.java

```diff
@@ -77,87 +74,73 @@
 
     public CassandraQueryExecutor(Configuration configuration)
     {
-        cluster = Cluster.builder()
-                .addContactPoint(configuration.getStringMandatory("databases.cassandra.host"))
-                .withPort(configuration.getIntMandatory("databases.cassandra.port"))
+        session = CqlSession.builder()
+                .addContactPoint(new InetSocketAddress(configuration.getStringMandatory("databases.cassandra.host"), configuration.getIntMandatory("databases.cassandra.port")))
+                .withLocalDatacenter("datacenter1")
```

Comment:
I think we shouldn't hard-code this value. How about retrieving from configuration?

Reply:
Good point. I introduced the optional property `databases.cassandra.local_datacenter` for this purpose.